### PR TITLE
cleanup: also cleanup taps.

### PR DESCRIPTION
**Pull request by mikemcquaid**
Let the cleanup step also uninstall taps that have been tapped.

CC @xu-cheng

### CODE REVIEW

File: lib/brewdler/commands/cleanup.rb [hunk 1]
```diff
@@ -2,13 +2,26 @@ module Brewdler::Commands
...
-        Kernel.system "brew", "uninstall", "--force", *formulae
-        puts "Uninstalled #{formulae.size} formula#{ formulae.size == 1 ? "" : "e"}"
+        if formulae.any?
+          Kernel.system "brew", "uninstall", "--force", *formulae
+          puts "Uninstalled #{formulae.size} formula#{ formulae.size == 1 ? "" : "e"}"
+        end
+
+        if taps.any?
+          Kernel.system "brew", "untap", *taps_to_untap
+        end
```
Comment (xu-cheng) [inline lib/brewdler/commands/cleanup.rb]: Maybe a `puts "Untapped ..."` here

**Comment by xu-cheng** [comment]
Need to fix the test, otherwise LGTM :+1:

File: lib/brewdler/commands/cleanup.rb (hunk 1 above)
Comment (xu-cheng) [inline lib/brewdler/commands/cleanup.rb]: Also I think it should be `Kernel.system "brew", "untap", *taps`

Comment (mikemcquaid) [inline lib/brewdler/commands/cleanup.rb]: `brew untap` outputs the `Untapped ...`.

Comment (mikemcquaid) [inline lib/brewdler/commands/cleanup.rb]: And yeh, good spot, thanks.

**Comment by xu-cheng** [comment]
I have help to fix the test.

**Comment by mikemcquaid** [comment]
I :heart: you @xu-cheng :wink:

**Comment by mikemcquaid** [comment]
@xu-cheng You happy for this to be merged?

**Comment by xu-cheng** [comment]
Yeah, I will rebase it then do the merge.

**Pull request closed without merging**